src/main/resources/default/assets/common/core.js

```diff
@@ -1,11 +1,11 @@
-// Checks if the given string is empty or null.
-sirius.isEmpty = function (aString) {
-    return aString === null || aString === undefined || aString === '';
+// Checks if the given value is empty or null.
+sirius.isEmpty = function (value) {
+    return value === null || value === undefined || typeof value === 'undefined' || value === '';
```

Comment:
```suggestion
    return value === null || typeof value === 'undefined' || value === undefined || value === '';
```
i think they need to be flipped to prevent some errors